fix #179: guess the zone type also for reverse zones only

lisuml opened the pull request.

lisuml committed:
fix #179: guess the zone type also for reverse zones only

apatard commented:
too bad. I've only noticed now this change, when working on trying to submit my patch for the same problem.
I've been using a different patch for quite some time: https://github.com/apatard/ansible-role-bind/commit/324f829843e38f7c1fc893bcc260dfdb79c34b04 (don't rely on the dates).
Not sure which one is the best. In the end, what matters is that a fix is merged.

docsvt committed:
Eliminate dnssec-enable option for bind version 9.16 and newer.

In bind version 9.16.0 the dnssec-enable option was made obsolete
and in 9.18.0 the option was entirely removed. Patch check
installed bind version and set variable bind_dnssec_enable
to false if version 9.16 and newer. Since the name of the package
is different in operating systems families i added variable bind_package.

docsvt committed:
Add cgroupns_mode for molecule

Commit:
Merge remote-tracking branch 'docsvt/obsoleted_dnssec_enable'
